test/eutils.jl

```diff
@@ -4,22 +4,22 @@
         res = einfo(db="pubmed")
         @test res.status == 200
         @test startswith(res.headers["Content-Type"], "text/xml")
-        @test isa(parsexml(res.data), EzXML.Document)
+        @test isa(parsexml(res.body), EzXML.Document)
     end
 
     @testset "esearch" begin
         res = esearch(db="pubmed", term="asthma")
         @test res.status == 200
         @test startswith(res.headers["Content-Type"], "text/xml")
-        @test isa(parsexml(res.data), EzXML.Document)
+        @test isa(parsexml(res.body), EzXML.Document)
```

Comment:
So fo instance, here we could check the number of articles it returns. Also I suggest using a more complex 'term' -- issues may appear when escaping complex terms e.g. """(Asthma[MeSH Major Topic]) AND ("1/1/2018"[Date - Publication] : "3000"[Date - Publication])"""